app/src/main/java/io/homeassistant/companion/android/sensors/LightSensorManager.kt

```diff
@@ -82,16 +82,14 @@ class LightSensorManager : SensorManager, SensorEventListener {
     }
 
     override fun onSensorChanged(event: SensorEvent?) {
-        if (event != null) {
-            if (event.sensor.type == Sensor.TYPE_LIGHT) {
-                onSensorUpdated(
-                    latestContext,
-                    lightSensor,
-                    event.values[0].roundToInt().toString(),
-                    "mdi:brightness-5",
-                    mapOf()
-                )
-            }
+        if (event != null && event.sensor.type == Sensor.TYPE_LIGHT) {

```

Comment:
Think this is even better?
```suggestion
        if (event?.sensor.type == Sensor.TYPE_LIGHT) {

```